src/pages/ConsumerClientManagePage/ConsumerClientManage.page.tsx

```diff
@@ -19,12 +18,15 @@ import SyncIcon from '@mui/icons-material/Sync'
 import { useDrawerState } from '@/hooks/useDrawerState'
 import { SetClientAdminDrawer } from './components/SetClientAdminDrawer/SetClientAdminDrawer'
 import { apiV2GuideLink } from '@/config/constants'
+import { useNavigate } from '@/router'
 
 const ConsumerClientManagePage: React.FC = () => {
   const { t } = useTranslation('client', { keyPrefix: 'edit' })
+  const { t: actionT } = useTranslation('common', { keyPrefix: 'actions' })
```

Comment:
usually we put "t" as prefix  and put name of file (instead of key prefix) 
```suggestion
  const { t: tCommon } = useTranslation('common', { keyPrefix: 'actions' })
```